index.html

```diff
@@ -2713,7 +2713,913 @@ <h5>Question</h5>
+                      <li>
+                          Fully private is possible.
+                      </li>
+                      <li>
+                          VDR is visible to "all", but "all" is a restricted audience with enforced terms of service or other disincentives to abuse.
+                      </li>
+                      <li>
+                          VDR is fully public with no constraints.
+                      </li>
+                  </ol>
+              </section>
+              <section>
+                  <h5>Relevance</h5>
+                  <p>
+                      Restricting the audience for a DID is a way to discourage crawling and secondary, possibly abusive publication. However, no mechanism can completely prevent this; creating disincentives, accountability, and/or costs that encourage the DID owner's wishes to be respected is the best we can hope for.
+                  </p>
+              </section>
+              <section>
+                  <h5>Examples</h5>
+                  <table class="simple">
+                      <tr>
+                          <th>
+                              Method
+                          </th>
+                          <th>
+                              Net.
+                          </th>
+                          <th>
+                              Reg.
+                          </th>
+                          <th>
+                              Notes
+                          </th>
+                      </tr>
+                      <tr>
+                          <td>
+                              did:foo
+                          </td>
+                          <td>
+                              A
+                          </td>
+                          <td>
+                              D
+                          </td>
+                          <td>
+                              By design, did:foo is super private.
+                          </td>
+                      </tr>
+                  </table>
+              </section>
+          </section>
+          <section>
+              <h4>Cross-DID Leakage</h4>
+              <section>
+                  <h5>Question</h5>
+                  <p>
+                      How possible is it to control multiple DIDs, without having an observer of one DID be able to deduce that another DID has the same controller?
+                  </p>
+              </section>
+              <section>
+                  <h5>Responses</h5>
+                  <ol type="A">
+                      <li>
+                          Each DID is entirely isolated; there is no practical way to infer relationships of common control between them.
+                      </li>
+                      <li>
+                          It may be possible to infer relationships of common control based on timing and the IP address of clients, but not based on data persisted in the VDR itself. Inference is expensive or impractical.
```

Comment:
```suggestion
It may be possible to infer relationships of common control based on
timing and the IP address of clients, but not based on data persisted
in the VDR itself. Inference is expensive or impractical.
```